features/networking/pod.feature

```diff
@@ -58,21 +58,25 @@ Feature: Pod related networking scenarios
   Scenario: The openflow list will be cleaned after delete the pods
     Given I have a project
     Given I have a pod-for-ping in the project
-    Then I use the "<%= pod.node_name(user: user) %>" node
+    Then evaluation of `pod.node_name(user: user)` is stored in the :node_name clipboard
```

Comment:
`(user: user)` shouldn't be needed. it is the default anyway.

Reply:
Thanks Akostadinov, removed (user: user).